.github/workflows/ci.yml

```diff
@@ -0,0 +1,90 @@
+name: CI
+
+on:
+  push:
+    branches: [ master ]
+  pull_request:
+    branches: [ master ]
+
+defaults:
+  run:
+    shell: bash # Windows' default shell is PowerShell
+
+jobs:
+  build:
+    runs-on: ${{ matrix.os }}
+    strategy:
+      fail-fast: false # when true, one job's failure will cancel other jobs
+      matrix:
+        os: [ ubuntu-16.04, macOS-latest, windows-latest ]
+    steps:
+      - name: Add "$HOME/.local/bin/" to PATH
+        run: echo "$HOME/.local/bin/" >> $GITHUB_PATH
+
+      # We set LF endings so that the Windows environment is consistent with Appveyor
```

Comment:
Why is it desirable to have the Windows environment consistent with AppVeyor? We haven't used AppVeyor in this repo's CI for quite a while.

Reply:
I was going to ask about that. Spago uses [this in their CI](https://github.com/purescript/spago/blob/master/.github/workflows/build.yml#L16-L21) likely because they were using Appveyor. I wasn't sure whether we needed it here or not.

I was thinking we should drop it.

Reply:
Ah, I see. Yeah I think dropping it sounds good if it doesn't cause any problems.

Reply:
Yeah we also have Appveyor running as part of the Spago CI, mostly as a smoke test to see if the new GH Actions CI was behaving as it should have. This has (surprisingly to me) caught quite a few inconsistencies and quirks actually.
As you can see it needs some special care to make the environments consistent, and we'll probably drop it from there in some time

Reply:
> This has (surprisingly to me) caught quite a few inconsistencies and quirks actually.

Such as?

Reply:
I don't recall the details unfortunately, nor I'm able to find the relevant commits right now. I suspect most of that was resolved in the big initial effort. In any case, one of such occurrences was e.g. that I forgot the `--pedantic` flag in GH Actions, so a build-breaking change failed AppVeyor, while the Actions were happily green.

I'm not advocating for running an AppVeyor build of course, I'm just saying that since Travis and AppVeyor were consistent, keeping AppVeyor as a reference point helped keep such consistency when moving to Actions.

So for this reason here we might want to run Travis alongside the new Action workflow, at least for some time.